stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -455,7 +455,6 @@ internal class ChatDomainImpl internal constructor(
             object : SocketListener() {
                 override fun onConnected(event: ConnectedEvent) {
                     storeBgSyncData()
```

Comment:
Is `ChatClient` is properly initialized in the Application class, then we might want this call to be executed multiple time when user changes account.